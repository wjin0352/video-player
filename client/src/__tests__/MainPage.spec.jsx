import React from 'react';
import { shallow } from 'enzyme';
import MainPage from '../components/main_page';

test('MainPage renders correctly', () => {
  const mainPage = shallow(<MainPage/>)
  expect(mainPage).toMatchSnapshot();
})