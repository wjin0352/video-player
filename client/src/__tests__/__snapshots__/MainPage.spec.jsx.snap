// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainPage renders correctly 1`] = `
<div
  className="main_page"
>
  <SearchBar
    onSearchTermChange={[Function]}
  />
  <container
    className="videos_container z-depth-5"
  >
    <Connect(VideoDetail)
      video={null}
    />
    <VideoList
      onVideoSelect={[Function]}
      videos={Array []}
    />
    <div />
  </container>
</div>
`;
